tests/unit/plugins/test_go.py

```diff
@@ -110,6 +66,30 @@ def test_schema(self):
                 "Expected {!r} to be included in properties".format(expected),
             )
 
+        # Check go-channel
+        go_channel = properties["go-channel"]
+        for expected in ["type", "default"]:
+            self.assertTrue(
+                expected in go_channel,
+                "Expected {!r} to be included in 'go-channel'".format(expected),
```

Comment:
note that here we're checking the string for single quotes.... (to be continued)